### PR TITLE
update swarm startup for tracing and metrics

### PR DESCRIPTION
**Pull request by nonsense**
This PR is adding support for metrics and tracing within the test swarm cluster.

`make cluster` is now starting the metrics and tracing setup, which are now part of the Terraform templates.

### CODE REVIEW

File: swarm-test-cluster/main.tf [hunk 1]
```diff
@@ -105,6 +105,12 @@ resource "docker_container" "swarm" {
     "--pprof",
     "--pprofaddr=0.0.0.0",
     "--ws",
+    "--tracing",
+    "--tracing.endpoint=172.18.0.1:6831",
```
Comment (nonsense) [inline swarm-test-cluster/main.tf]: This is really brittle, assumes we have started `jaeger` on the `swarm` network.

File: swarm-test-cluster/main.tf [hunk 2]
```diff
@@ -105,6 +105,12 @@ resource "docker_container" "swarm" {
...
+    "--tracing.svc=swarm${count.index+1}",
+    "--metrics",
+    "--metrics.influxdb.export",
+    "--metrics.influxdb.endpoint=http://172.19.0.1:8086",
```
Comment (nonsense) [inline swarm-test-cluster/main.tf]: This is also brittle, assumes we have started `stateth` after the swarm cluster and jaeger.

**Comment by nonsense** [comment]
@janos I've added support for configurable docker network on `stateth`, and also changed the docker run for jaeger to use the `swarm` network.

So by default this configuration should run nicely. Obviously if one starts to mess with variables, it can be broken, but I think it is nice that we have metrics and tracing out of the box.

**Comment by janos** [comment]
Thanks Anton, this looks pretty nice. The only thing that may be changed is the default number of swarm containers, 20 may be too much for the default.

**Pull request merged**